app/src/main/java/com/github/polypoly/app/menu/JoinGameLobbyActivity.kt

```diff
@@ -567,6 +566,6 @@ class JoinGameLobbyActivity : ComponentActivity() {
 private val authenticated_user = User(7, "current_user", "", Skin(0, 0, 0),
     Stats(0, 0, 0, 0, 0), listOf(), mutableListOf()
 )
-private val mockDb = FakeRemoteStorage()
+private val mockDb = com.github.polypoly.app.network.FakeRemoteStorage()
```

Comment:
Could probably be left out (tested and compiles)

Reply:
Fixed !